src/main/java/org/cryptimeleon/craco/sig/sps/kpw15/MatrixUtility.java

```diff
@@ -1,42 +1,41 @@
 package org.cryptimeleon.craco.sig.sps.kpw15;
 
-import org.cryptimeleon.math.structures.cartesian.Vector;
 import org.cryptimeleon.math.structures.groups.GroupElement;
 import org.cryptimeleon.math.structures.groups.cartesian.GroupElementVector;
 import org.cryptimeleon.math.structures.groups.elliptic.BilinearMap;
 import org.cryptimeleon.math.structures.rings.zn.Zp.ZpElement;
 
 /**
- * Performs the simple matrix operations required by the KPW15 SPS scheme.
- * */
+ * Performs a limited set of matrix operations required by the KPW15 SPS scheme.
+ */
 public class MatrixUtility {
```

Comment:
would like to move this in the KPW15 scheme class as an inner class. Because nobody should see it as a standalone matrix util.

Reply:
fair enough. And addresses my worry that someone might mistake this for a "proper" matrix utility.

Reply:
super.

Reply:
063bf1a646430ad1d16a50429c399165fffb7b27 for reference fixes it.